recipes/scikit-criteria/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "scikit-criteria" %}
+{% set version = "0.2.11" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/scikit-criteria-{{ version }}.tar.gz
+  sha256: 82274adbd468914e95be46bdf9322e772d3446b7a3bb6309e9aacd6a7ff5098f
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - attrs
+    - joblib
+    - json_tricks
+    - matplotlib-base
+    - numpy
+    - pulp
+    - python >=3.6
+    - scipy
+    - tabulate
+
+test:
+  imports:
+    - skcriteria
+    - skcriteria.datasets
+  commands:
+    - pip check
+  requires:
+    - pip
+    - mock  # for `pip check`
+    - pytest  # for `pip check`
+
+about:
+  home: http://scikit-criteria.org/
+  summary: Scikit-Criteria is a collections of algorithms, methods and
```

Comment:
This sentence appears too be incomplete

Reply:
Nice catch. Fixed by https://github.com/conda-forge/scikit-criteria-feedstock/pull/1

Reply:
Thanks Hadrien! 😄